src/content/language/libraries.md

```diff
@@ -89,29 +89,24 @@ import 'package:lib1/lib1.dart' show foo;
 import 'package:lib2/lib2.dart' hide foo;
 ```
 
-<a id="deferred-loading"></a>
-#### Lazily loading a library
+#### Lazily loading a library {:#lazily-loading-a-library}
```

Comment:
I wouldn't expect the header ID customization `{:#lazily-loading-a-library}` is needed in this case since it matches the original header. Any reason you needed to add it?